blueprintapi/views.py

```diff
@@ -2,7 +2,6 @@
 from rest_framework.decorators import api_view
 from rest_framework.response import Response
 
-
```

Comment:
Why'd you remove this space? Two blank spaces before a function or class definition is actually preferred: https://peps.python.org/pep-0008/#blank-lines

Reply:
Ah, sloppy removal of a comment.  Thanks for the catch.